src/pages/UserPage/UserPage.jsx

```diff
@@ -227,162 +260,248 @@ const UserPage = () => {
+          </label>
+          <span>
+            <input
+              className="userPage__form-input"
+              type="password"
+              id="userPassword"
+              name="password"
+              disabled={!isEditable}
+              onChange={handleInputChange}
+              placeholder={user.password}
+              value={formData.password}
+            />
+            <RiEdit2Fill
+              onClick={handleEditClick}
+              className="userPage__form-icon"
+            />
+            {errors.password && (
+              <p className="restaurantAdminView__error">{errors.password}</p>
+            )}
+          </span>
+          {user.phone_numbers &&
+            user.phone_numbers.map((phoneNumber, index) => (
               <div key={phoneNumber.id_user_phone_number}>
-                <label className='userPage__form-label' htmlFor='userPhoneNumber'>{languageSelector(language, 'signInPhone')} {index + 1}</label>
+                <label
+                  className="userPage__form-label"
+                  htmlFor="userPhoneNumber"
+                >
+                  {languageSelector(language, "signInPhone")} {index + 1}
+                </label>
                 <span>
-                <input
-                  className='userPage__form-input'
-                  type='text'
-                  id='userPhoneNumber'
-                  name={`phone_numbers[${index}][phone_number]`}
-                  disabled={!isEditable}
-                  onChange={handleInputChange}
-                  placeholder={phoneNumber.phone_number}
-                  value={formData.phone_numbers[index].phone_number}
-                />
-                <RiEdit2Fill onClick={handleEditClick}  className='userPage__form-icon'/>
-                {errors.phoneNumber && <p className='restaurantAdminView__error'>{errors.phoneNumber}</p>}
-              </span>
-            </div>
+                  <input
+                    className="userPage__form-input"
+                    type="text"
+                    id="userPhoneNumber"
+                    name={`phone_numbers[${index}][phone_number]`}
+                    disabled={!isEditable}
+                    onChange={handleInputChange}
+                    placeholder={phoneNumber.phone_number}
+                    value={formData.phone_numbers[index].phone_number}
+                  />
+                  <RiEdit2Fill
+                    onClick={handleEditClick}
+                    className="userPage__form-icon"
+                  />
+                  {errors.phoneNumber && (
+                    <p className="restaurantAdminView__error">
+                      {errors.phoneNumber}
+                    </p>
+                  )}
+                </span>
+              </div>
             ))}
-            {user.addresses  && user.addresses.map((address, index)=>(
+          {user.addresses &&
+            user.addresses.map((address, index) => (
```

Comment:
It's recommended for dynamic mapping rendering: `!!user.addresses.map`.

Reply:
Resolved

Reply:
Please, consider the proposal fix, thx.

Reply:
np, approved.